woocommerce/woocommerce
# E2e pw/migrate api core tests to use playwright 34126

### All Submissions:






### Changes proposed in this Pull Request:


This PR is for the creation of api-core-tests in playwright. The following test groups were created:
coupons - closes 353-gh-woocommerce/woocommerce-quality
hello - closes 352-gh-woocommerce/woocommerce-quality
orders - closes 354-gh-woocommerce/woocommerce-quality
products - closes 355-gh-woocommerce/woocommerce-quality
refunds - closes 356-gh-woocommerce/woocommerce-quality
shipping - closes 357-gh-woocommerce/woocommerce-quality

### How to test the changes in this Pull Request:

1. Ensure test environment has been initialised correctly
2. Navigate to woocommerce/plugins/woocommerce
3. USE_WP_ENV=1
4. pnpm playwright test --config=tests/api-core-tests/playwright.config.js
5. 120 tests should execute successfully

### Other information:




### FOR PR REVIEWER ONLY:
